### DISCUSSION TITLE
Integrated brier score

### DISCUSSION THREAD

**Discussion by indranilmallick** [CamDavidsonPilon/lifelines#1311] [Ideas]
Lifelines is an amazing library with excellent documentation. 
Is there a way to calculate an integrated brier score to measure the performance of a model?

**Comment by CamDavidsonPilon** [comment]
Hi @indranilmallick - thanks for the kind words. 

A brier score isn't implemented, but I think it should be added!

**Comment by ds21m013** [comment]
Hi!
I fully agree with @indranilmallick - lifelines is really great. I did a comparision of several time-to-event software packages for my master thesis and found lifelines as the most valuable.
As long as you are just working with proportional hazard models, the C-index is sufficient to compare the model quality. But if you need to do add some interpretations about the model error, other metrics like the Brier score or Time-dependent ROC would be really helpful.
Has anyone already implemented a version of the Brier score for lifelines that he or she would like to share?
Best wishes! Andreas
